chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,6 +150,12 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
+            # we want disks that are multipath but not if they are holders
+            # of a multipath
+            if d.multipath?
```

Comment:
If that is true, then would we still need to test for `multipath?` ? Couldnt we just check `held_by_multipath?` instead and make the flow easier?

I think I was confused by the different test conditions that were introduced and their combinations as Adam mentioned.